src/v2/Apps/Artist/Routes/Works/index.tsx

```diff
@@ -21,14 +24,21 @@ export const WorksRoute: React.FC<WorksRouteProps> = props => {
     isClient &&
     get(artist, a => a.related.artistsConnection.edges.length, 0) > 0
 
+  const { user } = React.useContext(SystemContext)
+  const artistSeriesIsEnabled = userHasLabFeature(user, "Artist Series")
+
   return (
     <>
       <Box>
         <ArtistTopWorksRail artist={artist} />
       </Box>
 
       <Box>
-        <ArtistCollectionsRail artistID={artist.internalID} />
+        {artistSeriesIsEnabled ? (
+          <ArtistSeriesRail artist={artist} />
```

Comment:
This takes the place of the collections rail (which is lazily loaded, and now we get full SSR for the series).